tests/usb/usb.go

```diff
@@ -0,0 +1,116 @@
+package usb
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	expect "github.com/google/goexpect"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	v1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/kubecli"
+
+	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
+	"kubevirt.io/kubevirt/tests"
+	"kubevirt.io/kubevirt/tests/console"
+	"kubevirt.io/kubevirt/tests/decorators"
+	"kubevirt.io/kubevirt/tests/framework/kubevirt"
+	"kubevirt.io/kubevirt/tests/libwait"
+	"kubevirt.io/kubevirt/tests/util"
+)
+
+const (
+	failedDeleteVMI = "Failed to delete VMI"
+	cmdNumberUSBs   = "dmesg | grep -c idVendor=46f4"
+)
+
+var _ = Describe("[Serial][sig-compute][USB] host USB Passthrough", Serial, decorators.SigCompute, decorators.USB, func() {
+	var (
+		virtClient kubecli.KubevirtClient
+		config     v1.KubeVirtConfiguration
+	)
+
+	BeforeEach(func() {
+		virtClient = kubevirt.Client()
+		kv := util.GetCurrentKv(virtClient)
+		config = kv.Spec.Configuration
+
+		nodeName := tests.NodeNameWithHandler()
+		Expect(nodeName).ToNot(BeEmpty())
+
+		// Emulated USB devices only on c9s providers. Remove this when sig-compute 1.26 is the
+		// oldest sig-compute with test with.
+		// See: https://github.com/kubevirt/project-infra/pull/2922
+		stdout, err := tests.ExecuteCommandInVirtHandlerPod(nodeName, []string{"dmesg"})
+		Expect(err).ToNot(HaveOccurred())
+		if strings.Count(stdout, "idVendor=46f4") == 0 {
+			Skip("No emulated USB devices present for functional test.")
+		}
+	})
+
+	AfterEach(func() {
```

Comment:
IIRC, I added this to reset the config. It was a [suggestion](https://github.com/kubevirt/kubevirt/pull/10015#discussion_r1251546630) from @alicefr, due the test use of feature gates.

Reply:
@xpivarc is the KubeVirt CR restored automatically? I don't have the impression as I mentioned in the comment linked by Victor

Reply:
https://github.com/kubevirt/kubevirt/blob/main/tests/utils.go#L1840

Reply:
@xpivarc ack, sorry, I missed it. Would it be fine to do it in a follow-up? I can try to find other [tests](https://github.com/kubevirt/kubevirt/blob/main/tests/vmi_hostdev_test.go#L39-L52) that do the same and submit a cleanup. Me asking is just to avoid a respin for this. (if there are other comments to the PR, than I'll include this in the next iteration)